crates/core/tedge_agent/src/device_profile_manager/mod.rs

```diff
@@ -1,14 +1,23 @@
+use camino::Utf8PathBuf;
+use tedge_utils::file::create_file_with_defaults;
+use tedge_utils::file::FileError;
+
+pub struct DeviceProfileManagerBuilder {}
+
+impl DeviceProfileManagerBuilder {
+    pub fn try_new(ops_dir: &Utf8PathBuf) -> Result<Self, FileError> {
+        let workflow_file = ops_dir.join("device_profile.toml");
+        if !workflow_file.exists() {
+            let workflow_definition = r#"
```

Comment:
You can also use [include_str!](https://doc.rust-lang.org/std/macro.include_str.html):

```suggestion
            let workflow_definition = include_str!("device_profile.toml");
```

Reply:
Resolved by 4fedbabd8760b6c00565c3d4af6ccc145cb34d63